tracking/load_experimental_data.py

```diff
@@ -5,39 +5,37 @@
 
 
 class C3dData:
-    def __init__(self, file_path):
+    """
+    The base class for managing c3d file
+
+    Attributes
+    ----------
+    c3d: ezc3d.c3d
+        The c3d file
+    marker_names: list
+        The list of all marker names in the biorbd model.
+    trajectories: ndarray
+        The position of the markers
+    Methods
+    -------
+    get_marker_trajectories(loaded_c3d: c3d, marker_names: list) -> np.ndarray
+        Get markers trajectories
+    get_indices(self)
+        Get the indices of start and end
+    get_final_time(self)
+        Get the final time of c3d
+    """
+
+    def __init__(self, file_path: str, biorbd_model: biorbd.Model):
         self.c3d = c3d(file_path)
-        self.marker_names = [
-            "MAN",  # check 1
-            "XYP",  # check 2
-            "C7",  # check 3
-            "T10",  # check 4
-            "CLAV_SC",  # check 5
-            "CLAV_AC",  # check 6
-            # "SCAP_Cor",  # check 7
-            "SCAP_IA",  # check 8
-            # "SCAP_AA",  # check 9
-            "SCAP_AC",  # check 10
-            "SCAP_BACK",  # check 11
-            "SCAP_FRONT",  # check 12
-            "EPI_lat",  # check 13
-            "EPI_med",  # check 14
-            "DELT",  # check 15
-            "ARM",  # check 16
-            "ULNA",  # check 17
-            "ELB",  # check 18
-            "RADIUS",  # check 19
-            "SEML",  # check 20
-            "MET2",  # check 21
-            "MET5",  # check 22
-        ]
-
+        self.marker_names = [biorbd_model.markerNames()[i].to_string() for i in range(len(biorbd_model.markerNames()))]
         self.trajectories = self.get_marker_trajectories(self.c3d, self.marker_names)
 
     @staticmethod
-    def get_marker_trajectories(loaded_c3d, marker_names):
+    def get_marker_trajectories(loaded_c3d: c3d, marker_names: list) -> np.ndarray:
```

Comment:
add self as an argument,
remove c3d and use self.c3d in the function
marker_names is None by default.
this is not a staticmethod